woocommerce-services.php

```diff
@@ -788,8 +788,16 @@ private function dismiss_tos_notice() {
 		 * @return bool
 		 */
 		private function can_accept_tos() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG && current_user_can( 'manage_options' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
```

Comment:
It might be overly defensive now, but isn't harming anything and allows for some flexibility for calling this function elsewhere.